### DISCUSSION TITLE
Master_template Inhalte übersteuern

### DISCUSSION THREAD

**Discussion by nicole-vlbg** [e-netze/webgis-community#137] [Q&A]
Kann man im master_template voreingestellte Inhalte, in diesem Fall Druckvorlagen übersteuern, wenn sie in einer Speziellen Karte nicht verwendet werden bzw. nur andere auf dieses Kartenthema angepasste Vorlagen verwendet werden sollen?

Wenn ja, wie ist das möglich?

![image](https://github.com/gis-eni/webgis-forum/assets/124671123/1d10e76c-2603-4e47-ab2e-cc255f4b8594)

**Comment by gis-eni** [comment]
Geht denk ich leider nicht. Das Master Template ist dazu da, dass man Inhalte für alle Karten einstellt. Das man damit in Spezialkarten wieder etwas wegnimmt ist nicht vorgesehen.
Oder verstehe ich die Frage falsch?

**Reply by ue-black** [reply]
Nein, das hast du ganz richtig verstanden. Das Problem ist nur, dass wir in 49 Karten die 4 Standard Drucklayouts brauchen und in einer Karte diese nicht verwenden wollen. Wir dachten viell. gibt es da einen Trick wie man das trotz Master template lösen kann. In den einzelnen Karten sind ja keine Drucklayouts ausgewählt, wir dachten wenn man welche auswählt (siehe oben die Layouts für den FWP) könnte man die Standardlayouts deaktivieren oder werden diese dann nicht dargestellt (man sieht sie ja trotzdem im Mapbuilder aufgelistet).

**Comment by gis-eni** [comment]
Verstehe. Nur wie gesagt, die Master Vorlagen sind dazu da, dass man gewisse Einstellungen in alle Karten bringen kann. Wir haben das eingeführt, weil es immer wieder "neue" Dinge (zum Beispiel ein Standarddrucklayout) gibt, das in alle Karten rein muss. Und damit man nicht immer alle 49 Karten angreifen muss, stellt man das übergeordnet ein.

Das Verhalten zu ändern wäre leider ein Breaking Change. Da könnte man vieles bei bestehenden Installation kaputt machen...

Die Master ist allerdings immer nur für eine Portalseite gültig (gehe zumindest davon aus ;))
Darum gibt es für diesen einen Fall einen Workaround:

* Mach eine neue Portal Seite
* In der erstellst du dann deine "Spezialkarte"

Dann erscheint diese Karte allerdings nicht mehr auf der Startseite des Portals. Wenn du die Karte sowieso nur über einen anderen Link aufrufst ist das egal, wenn nicht, gibts auf dafür einen Workaround:

* Erstelle auf der eigentlich Portalseite eine App mit dem AppBuilder (typ redirect)
* Dort gibst du die Url zu der neuen Karte an (siehe https://docs.webgiscloud.com/cloud/webgis_apps/appbuilder/index.html)
* Auf der Portalseite erscheint jetzt eine Kachel, die dich zu Spezialkarte weiter leitet.

**Reply by ue-black** [reply]
Klingt gut, schauen wir uns an. Danke!
